spec/factories.rb

```diff
@@ -435,27 +436,27 @@ def r_str
   factory(:federation_person_from_webfinger, class: DiasporaFederation::Entities::Person) do
     sequence(:guid) { UUID.generate :compact }
     sequence(:diaspora_id) {|n| "bob-person-#{n}#{r_str}@example.net" }
-    url "https://example.net/"
-    exported_key OpenSSL::PKey::RSA.generate(1024).public_key.export
+    url { "https://example.net/" }
+    exported_key { OpenSSL::PKey::RSA.generate(1024).public_key.export }
     profile {
       DiasporaFederation::Entities::Profile.new(
-        FactoryGirl.attributes_for(:federation_profile_from_hcard, diaspora_id: diaspora_id))
+        FactoryBot.attributes_for(:federation_profile_from_hcard, diaspora_id: diaspora_id))
```

Comment:
Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.